src/main/fc/runtime_config.h

```diff
@@ -40,11 +40,13 @@ typedef enum {
     ARMING_DISABLED_CMS_MENU    = (1 << 7),
     ARMING_DISABLED_OSD_MENU    = (1 << 8),
     ARMING_DISABLED_BST         = (1 << 9),
+    ARMING_DISABLED_NO_GYRO     = (1 << 10),
```

Comment:
This should probably go to the top of the list. The intention is to have the list ordered by importance, and use beep codes to indicate the most important reason.

(Also, please add a comment to the list indicating this.)